packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -95,8 +95,6 @@ class _ScaffoldLayout extends MultiChildLayoutDelegate {
   }
 }
 
-final _ScaffoldLayout _scaffoldLayout = new _ScaffoldLayout();
```

Comment:
if it's always going to return true, then you can keep on sending the same instance each time